src/library/scala/collection/convert/LongAccumulator.scala

```diff
@@ -130,19 +147,13 @@ final class LongAccumulator
   }
 
   /** Retrieves the `ix`th element, using an `Int` index. */
-  final def apply(i: Int): Long = apply(i.toLong)
-
-  /** Returns a `LongStepper` over the contents of this `LongAccumulator`. */
-  final def stepper: LongStepper = new LongAccumulatorStepper(this)
+  def apply(i: Int): Long = apply(i.toLong)
```

Comment:
I can't see how any good could possibly come of this being non-final.  It just opens up the possibility of apply not matching itself!

Reply:
(Same as above, the class is final)